examples/precision_recall_at_k.py

```diff
@@ -0,0 +1,73 @@
+"""
+This module illustrates how to compute Precision at k and Recall at k metrics.

```

Comment:
I'd rather stop here for the module description and provide more details in the FAQ entry (plus, it says that we "predict all
 +the ratings for the pairs (user, item) that are not in the training set", which is not the case)